### PR TITLE
Add a Task table

### PR DESCRIPTION
**Pull request by jSylvestre**
Issue #223 
Just the tables

### CODE REVIEW

File: Harvest.Core/Domain/Ticket.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        public int ProjectId { get; set; }
+        public Project Project { get; set; }
+        public int? InvoiceId { get; set; }
+        public Invoice Invoice { get; set; }
+
+        [Required]
+        public int CreatedById { get; set; }
+        public DateTime CreatedOn { get; set; }
+
+        public int UpdatedById { get; set; }
```
Comment (srkirkland) [inline Harvest.Core/Domain/Ticket.cs:24]: You'll need to make this int? For it to be optional

File: Harvest.Core/Domain/Ticket.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        public DateTime UpdatedOn { get; set; }
```
Comment (srkirkland) [inline Harvest.Core/Domain/Ticket.cs:25]: Same here. Datetime needs to be nullable to be optional

File: Harvest.Core/Domain/Ticket.cs [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+        [StringLength(50)]
+        [Display(Name = "Task Name")]
+        public string Name { get; set; }
+        [Required]
+        public string Requirements { get; set; }
+        public DateTime DueDate { get; set; }
```
Comment (srkirkland) [inline Harvest.Core/Domain/Ticket.cs:32]: Maybe this should be optional too? Things might not always have a due date

File: Harvest.Core/Domain/Ticket.cs [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+        public string WorkNotes { get; set; } //For the person doing the Task
```
Comment (srkirkland) [inline Harvest.Core/Domain/Ticket.cs:34]: Probably want to make this a Max length string

**Review by srkirkland — CHANGES_REQUESTED** [review]

File: Harvest.Core/Domain/Ticket.cs (hunk 4 above)
Comment (jSylvestre) [reply Harvest.Core/Domain/Ticket.cs:36]: It defaults to max. Doesn't seem to be a simple way of explicitly doing it.

Comment (srkirkland) [reply Harvest.Core/Domain/Ticket.cs:36]: i would think you could just do [MaxLength(int.Max)]` or in the prop.  not sure if there's a better way

**Review by srkirkland — APPROVED** [review]

**Pull request merged**